typify-impl/src/lib.rs

```diff
@@ -205,21 +205,28 @@ pub struct TypeSpaceSettings {
 /// Per-type modifications.
 #[derive(Debug, Default, Clone)]
 pub struct TypeSpacePatch {
+    /// Replacement name
     rename: Option<String>,
+    /// Additional derives
     derives: Vec<String>,
 }
 
-/// By-name replacements
+/// By-name replacements.
 #[derive(Debug, Default, Clone)]
 pub struct TypeSpaceReplace {
+    /// Replacement name
     replace_type: String,
+    /// Not implemented
```

Comment:
apologies if this is implemented and I havent found/understood it.  If so, pls point me at it.

Reply:
It's used to indicate which of the impls we care about are provided by the given type.

Reply:
It's used to indicate which of the impls we care about are provided by the given type.

Reply:
Could you point me to the relevant code where these members are used.

Reply:
https://github.com/oxidecomputer/typify/blob/main/typify-impl/src/lib.rs#L404
https://github.com/oxidecomputer/typify/blob/main/typify-impl/src/lib.rs#L340
https://github.com/oxidecomputer/typify/blob/main/typify-impl/src/type_entry.rs#L526